src/components/NewMessage/NewMessage.vue

```diff
@@ -408,7 +433,11 @@ export default {
 
 		isMobileDevice() {
 			return /Android|iPhone|iPad|iPod/i.test(navigator.userAgent)
-		}
+		},
+
+		chatInput() {
```

Comment:
If i remember, we should be able to come back to typing normal message after editing something
so we need to have in store state for each: `chatInput` and `chatEditInput`

Reply:
Yes, it makes sense, I will add it in the description.